src/vivarium_ciff_sam/data/loader.py

```diff
@@ -442,8 +446,8 @@ def load_therapeutic_zinc_rr(key: str, location: str) -> pd.Series:
 
     return convert_to_dichotomous_rr(
         exposed_rr,
-        data_keys.PREVENTATIVE_ZINC.AFFECTED_ENTITY,
-        data_keys.PREVENTATIVE_ZINC.AFFECTED_MEASURE
+        data_keys.THERAPEUTIC_ZINC.AFFECTED_ENTITY,
+        data_keys.THERAPEUTIC_ZINC.AFFECTED_MEASURE
```

Comment:
Oh no; I'm sorry about these.This seems like something I should've caught in validation; does that mean my validation was off and I didn't notice?

Reply:
Nope, this was an issue I introduced.